test/index.test.js

```diff
@@ -132,6 +132,49 @@ describe('guardian.js', function () {
         });
       });
 
+      describe('when no transaction is returned on reply', function () {
```

Comment:
This condition should not happen: https://github.com/auth0/auth0-guardian.js/blob/master/lib/utils/http_client.js#L77 if the response is not ok then it returns an error, if it is ok it should contain the token. Anyway this would help better trace back the error if it happens because of this.

Reply:
I changed it so that the condition is that 'jwtToken throws an error', and used a mocked jwtToken to throw it.